string joint_name
float64 command
float64 error
float64 output